template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/queryprocessor/QueryStoreRoles.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders.queryprocessor;
+
+public class QueryStoreRoles {
+    // Service name.
+    public static final String QUERY_PROCESSOR = "QUERY_PROCESSOR";
+
+    // Roles
+    public static final String QUERYPROCESSOR = "QUERY_PROCESSOR";
```

Comment:
the value is the same

Reply:
I cannot see any difference, could you please take a look you local git commits?